cli/lib/tasks/cache.js

```diff
@@ -15,15 +27,51 @@ const clear = () => {
 
 const list = () => {
   return getCachedVersions()
-  .then((versions) => {
-    logger.log(versions.join(', '))
+  .then((binaries) => {
+    const table = new Table({
+      head: [a('version'), a('last used')],
```

Comment:
I would probably opt for white (I think this is default) or grey text here. The red does not look good visually 😅. Every other use case of `chalk.red` that I found also denotes errors/warnings/failures so seems inappropriate functionality-wise.